astropy_helpers/commands/build_ext.py

```diff
@@ -255,7 +256,6 @@ def finalize_options(self):
 
             extensions = self.distribution.ext_modules
             if extensions:
-                build_py = self.get_finalized_command('build_py')
```

Comment:
Is it really the same thing as `setuptools.command.build_py` ? I think it does more than that,  `setuptools.command.build_py` is just the class. But I don't know well the setuptools/distutils internals.